docs/migration-v8.md

```diff
@@ -422,6 +422,9 @@ IAsyncPolicy<HttpResponseMessage> asyncPolicyT = Policy.RateLimitAsync<HttpRespo
 
 ### Rate limit in v8
 
+> [!NOTE]
+> In v8, you have to add the `Polly.RateLimiting` package to your application otherwise you won't see the `AddRateLimiter` extension.
```

Comment:
```suggestion
> In v8, you have to add the [`Polly.RateLimiting`](https://www.nuget.org/packages/Polly.RateLimiting) package to your application otherwise you won't see the `AddRateLimiter` extension.
```

same for the others